src/robusta/cli/auth.py

```diff
@@ -22,8 +22,10 @@
 
 
 class RSAKeyPair(BaseModel):
-    prv: str
-    pub: str
+    prv: str = "none"
```

Comment:
I think the default value should be None and not "none"
For example:
prv: str = None

It's like python null, as opposed to string with the value "none"